### PR TITLE
pass-otp, pass-update: update to 1.2.0, 2.1

### PR DESCRIPTION
**Pull request by judaew**
#### Description

Update pass-otp (1.1.1 -> 1.2.0) and pass-update (1.0 -> 2.1)

###### Type(s)

- [ ] bugfix
- [x] update
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.6
Xcode 11.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: security/pass-otp/Portfile [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-checksums           rmd160  850dd63a85bbf7b92a091e75601c4e37c230b73a \
-                    sha256  d5137cffa4cb9dccb38765a340be109a9ae9350979045a1f449cf72a5c4e3c1c \
-                    size    46296
+checksums           rmd160  3ac479108fa4717af3142f7468e6abb2e57622f4 \
+                    sha256  5720a649267a240a4f7ba5a6445193481070049c1d08ba38b00d20fc551c3a67 \
+                    size    46789
 
-depends_run         port:pass \
-                    port:oath-toolkit
+depends_run         port:pass port:oath-toolkit port:qrencode
```
Comment (mf2k) [inline security/pass-otp/Portfile:24]: Please keep the style of one dependency per line. Also sort them alphabetically.

File: security/pass-update/Portfile [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
+github.tarball_from releases
+
 categories          security
-description         A pass extension that provides a convenient solution to update an existing password.
-long_description    ${description}
 platforms           darwin
 supported_archs     noarch
-license             GPL-2+
+license             GPL-2
+maintainers         {judaew.com:judaew @judaew} openmainntainer
```
Comment (mf2k) [inline security/pass-update/Portfile:13]: `openmaintainer` is spelled wrong here.

**Comment by judaew** [comment]
Done, thanks 🌟

**Comment by lpefferkorn** [comment]
Thanks for the update @judaew!

**Pull request merged**